src/main/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs.java

```diff
@@ -74,8 +72,6 @@
             doc="File to which variants should be written", optional=false)
     private File outputFile;
 
-    private static String GVCF_BLOCK = "GVCFBlock";
```

Comment:
At some point I had made the change to expose it in public, but then decided it wasn't worth making  separate pr to public and inlined it and forgot to reextract the constant.  Will fix..